packages/logging/src/log.js

```diff
@@ -591,14 +609,22 @@ Log.prototype.write = function(entry, options, callback) {
  *
  * @param {object} entry - An entry object.
  */
-Log.prototype.formatEntryForApi_ = function(entry) {
-  if (!(entry instanceof Entry)) {
-    entry = this.entry(entry);
-  }
+Log.prototype.decorateEntries_ = function(entries, callback) {
+  var self = this;
 
-  var formattedEntry = entry.toJSON();
-  formattedEntry.logName = this.formattedName_;
-  return formattedEntry;
+  async.map(entries, function(entry, callback) {
```

Comment:
Phew, good catch. I updated it to be a non-dev dependency. I'm not opposed to promises, though, I just wasn't sure how to write it. How would it look?